### ISSUE TITLE
Case of wrong indentation of |> in printed code

### ISSUE THREAD

**Issue by jindraivanek** [unisonweb/unison#2165]
This code

```
fixDigit i x = cases
    Sudoku digits ->
        digits |> List.mapIndexed (j -> cases (Digit s) ->
            if i == j then Digit (Set.fromList [x]) else Digit s
        ) |> Sudoku
```

gets printed as

```
fixDigit : Nat -> Nat -> Sudoku -> Sudoku
  fixDigit i x = cases
    Sudoku digits ->
      use Nat ==
         digits
      |> mapIndexed
           (j c3sjrdoffv1 ->
             (match c3sjrdoffv1 with
               Digit s -> if i == j then Digit (Set.fromList [x]) else Digit s))
      |> Sudoku
```

where both `|>` operators report error.

**Comment by hojberg** [comment]
As someone who uses `|>` quite a lot I've run into this as well

**Comment by atacratic** [comment]
related #1035

**Comment by pchiusano** [comment]
Fixed by #2399 

@runarorama fyi if you say “fixes #ticket1 and #ticket2” GitHub will only close #ticket1. The parser doesn’t seem to be very smart. 😀 You can do “fixes #ticket1, fixes #ticket2” instead.
